src/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated.cpp

```diff
@@ -105,6 +107,17 @@ Shared<const Frame> Tabulated::getFrame() const
     return accessFrame();
 }
 
+Array<Interpolator::Type> Tabulated::getInterpolationTypes() const
```

Comment:
I think this function is obsolete because we provide only a single interpolation type so there's no point returning an array.